web/blog/a-first-look-at-esql/index.md

```diff
@@ -0,0 +1,213 @@
+  filters the input with an expression. We have the same
+  [`where`](/operators/transformations/where) in TQL.
+- [`STATS`](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-stats-by.html):
+  groups its input via `BY` and applies aggregation functions on select fields
+  of each group.  Elastic went with Splunk nomenclature for this central
+  operations. Perhaps also to make the transition from Splunk to Elastic as easy
+  as possible.
+- [`ENRICH`](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-enrich.html):
+  adds data from existing indexes. It's effectively a join operation, and the
+  `ON` keywords makes it possible to select the join field. Interestingly, the
+  word "join" doesn't appear on the documentation. We hypothesize that this was
+  a conscious choice, as a database join may feel intimidating for beginning and
+  intermediate users.
+- [`KEEP`](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-keep.html):
+  selects a set of columns from the input and drops all others. It is the
+  inverse of
+  [`DROP`](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-drop.html).
+  In TQL, we call these projection operators
+  [`select`](/operators/transformations/select) and also
+  [`drop`](/operators/transformations/drop).
+- [`SORT`](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-sort.html):
+  sorts rows by one or more fields. `SORT height DESC, first_name ASC` sorts the
+  field `height` in descending order and the field `first_name` in ascending
+  order. The syntax of our [`sort`](/operators/transformations/sort) is
+  identical. Controlling the position of null values works with `NULLS FIRST`
+  and `NULLS LAST`. In TQL, we went Kusto-like with `nulls-first` and
+  `nulls-last`.
+- [`LIMIT`](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-limit.html):
+  restrict the number of output rows to a given number. In TQL, we have
+  [`head`](/operators/transformations/head) and
+  [`tail`](/operators/transformations/tail) for this purpose.
+
+[^1]:  In general, we find that Kusto has very self-descriptive operator names. During the design of TQL, we compared many different languages and often favored Kusto's choice of name.
+
+## Sources, Transformations, ... but Sinks?
+
+ES|QL differentiates two types of commands (aka operators):
+
+1. [Source commands](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-source-commands.html)
+2. [Processing commands](https://esql-latest.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-processing-commands.html)
+
+In TQL, an operator is a*source*, a *transformation*, or a *sink*. Some
+operators can be of multiple categories, like
```

Comment:
```suggestion
In TQL, an operator is a *source*, a *transformation*, or a *sink*. Some
operators can be of multiple categories, like
```

You could also use links here instead of italics to match the ES|QL categories.